lib/elixir/lib/kernel.ex

```diff
@@ -603,7 +603,8 @@ defmodule Kernel do
       -10
 
   """
-  @spec round(number) :: integer
+  @spec round(float) :: integer
+  @spec round(arg) :: arg when arg: integer
```

Comment:
based on the previous discussion, do you guys want me to rename `arg` to `value` ?